styles.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+    font-family: "Figtree-ExtraBold";
+    src: url('./assets/fonts/static/Figtree-ExtraBold.ttf') format('truetype');
+}
+
+@font-face {
+    font-family: "Figtree-Medium";
+    src: url('./assets/fonts/static/Figtree-Medium.ttf') format('truetype');
+}
+
+body {
+    background-color: rgb(244, 208, 78);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 100vh;
+}
+
+img {
+    border-radius: 10px;
+}
+
+h1, h2, p {
+    line-height: 150%;
+    letter-spacing: 0px;
+}
+
+h1 {
+    font-size: 24px;
+    font-family: "Figtree-ExtraBold", sans-serif;
+}
+
+h2 {
+    font-family: "Figtree-ExtraBold", sans-serif;
+}
+
+h2, p {
+    font-size: 16px;
+}
+
+p {
+    font-family: "Figtree-Medium", sans-serif;
+}
+
+.author {
+    display: flex;
+    align-items: center;
+}
+
+.tab {
+    display: flex;
+    padding: 0px;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.learning {
+    background-color: rgb(244, 208, 78);
+    border-radius: 5px;
+    padding: 5px 15px;
+    margin-bottom: 0;
+    font-size: 16px;
+}
+
+.wrapper-tab {
+    width: 384px;
```

Comment:
Don't need to change it in this project since you'll work a lot more on responsive web design soon, but wanted to point it out: this makes the card overflow on smaller screen widths (like mobile size). `max-width` instead of `width` should help

![Screenshot 2025-04-12 at 7 14 17 PM](https://github.com/user-attachments/assets/b86c87a0-cdb3-4bff-bcec-b50d8efba35a)